tests/test_twitterCleanup.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, mock
+import datetime
```

Comment:
I think that, overall, doing `from datetime import datetime` and `from unittest,mock import Mock, patch` would grant us better readability in this file.

Also, may we sort import in alphabetical order?

```
from datetime import datetime
from unittest import TestCase
from unittest.mock import Mock, patch
```